testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeOauth2HttpsProxy.java

```diff
@@ -42,7 +41,7 @@ public class EnvSinglenodeOauth2HttpsProxy
     @Inject
     public EnvSinglenodeOauth2HttpsProxy(DockerFiles dockerFiles, PortBinder binder, Standard standard, HydraIdentityProvider hydraIdentityProvider)
     {
-        super(ImmutableList.of(standard, hydraIdentityProvider));
+        super(standard, hydraIdentityProvider);
```

Comment:
I would leave it as it is and would open up a new PR which would fix it for all the environment. WDYT ?

Reply:
let's do it.